src/main/java/frc/robot/commands/ClimberCommands.java

```diff
@@ -42,9 +43,10 @@ public static Command unclimb(Subsystems subsystems) {
   }
 
   public static Command prepareToClimb(Subsystems subsystems) {
-    return Commands.parallel(
-        Commands.runOnce(() -> unclimb(subsystems), subsystems.climber),
+    return Commands.sequence(
+        Commands.print("PREPARE TO CLIMB"),
```

Comment:
We can remove the `Commands.print()` and outer `Commands.sequence()` now that we debugged the issue.